### PR TITLE
Pull request for WP-1463-country-setting

### PR DESCRIPTION
**Pull request by sduthil**
Depends-On: https://github.com/wazo-platform/wazo-bus/pull/110

## localization: validate country

### CODE REVIEW

**Comment by sduthil** [comment]
recheck

**Comment by sduthil** [comment]
recheck

File: wazo_confd/plugins/user/schema.py [hunk 1]
```diff
@@ -50,6 +50,7 @@ class UserSchema(BaseSchema):
     lastname = fields.String(validate=Length(max=128), allow_none=True)
     email = fields.Email(validate=Length(max=254), allow_none=True)
     timezone = fields.String(validate=Length(max=128), allow_none=True)
+    country = fields.String(dump_only=True)
```
Comment (pc-m) [inline wazo_confd/plugins/user/schema.py:53]: Should we modify the [unpaginated_user_strategy](https://github.com/wazo-platform/xivo-dao/blob/master/xivo_dao/resources/user/strategy.py#L7) to joinedload the country?

Comment (DrPyser) [inline wazo_confd/plugins/user/schema.py:53]: Should update wazo_confd/plugins/user/api.yml with `country` as `readOnly: true` too.

Comment (sduthil) [reply wazo_confd/plugins/user/schema.py:53]: Done, thanks

Comment (sduthil) [reply wazo_confd/plugins/user/schema.py:53]: Yes, we added the `tenant` table to the strategy: https://github.com/wazo-platform/xivo-dao/pull/278/commits/a96cb6fa0aef9f6c3d68ef83ad4ea6e45745fb92

Thanks for the comment :)

**Comment by sduthil** [comment]
Tests failed because dao and manage-db are half-merged, merging.

**Pull request merged**